apps/antalmanac/src/components/Map/Map.tsx

```diff
@@ -97,33 +99,33 @@ export default function CourseMap() {
     const [markers, setMarkers] = useState(getCoursesPerBuilding());
     const [calendarEvents, setCalendarEvents] = useState(AppStore.getCourseEventsInCalendar());
 
-    const updateMarkers = useCallback(() => {
-        setMarkers(getCoursesPerBuilding());
-    }, [setMarkers, getCoursesPerBuilding]);
-
-    const updateCalendarEvents = useCallback(() => {
-        setCalendarEvents(AppStore.getCourseEventsInCalendar());
-    }, [setCalendarEvents]);
-
     useEffect(() => {
+        const updateMarkers = () => {
+            setMarkers(getCoursesPerBuilding());
+        };
```

Comment:
Minor housekeeping to clean this up.

Reply:
Thx for these comments to explain why these changes are here. Although again, would be better in their own PR that is just for refactoring. A little git hack that might make it easier:

1. selectively stage only the refactor changes to your index
2. commit them as their own atomic commit
3. stash
4. revert the refactor commit
5. checkout to main, make a new branch, then reflog to find the id and cherry-pick that commit back onto your new branch
6. open new PR
7. checkout back to your feature branch and pop the stash back on

Now that I lay it out like that it sounds slightly like a pain in the ass, but if are experienced enough to not need to look up any of the commands it's pretty fast, and probably saves more time during review than it takes during writing.